doc/index.rst

```diff
@@ -66,7 +66,7 @@ Barracuda
 Overview
 ========
 
-:doc:`Contact info <ts_xml:index:master-csc-table:Barracuda>`
+:ref:`Contact info <ts_xml:index:master-csc-table:Barracuda>`
```

Comment:
I think this needs context. Having "Contact info" as the first thing in "Overview" is awkward.
Maybe at the bottom of the overview, we write, "As will all CSCs, information on the package, developers and product owners can be found in the `Master CSC Table <ts_xml:index:master-csc-table:Barracuda>` "